src/Image.php

```diff
@@ -64,7 +64,7 @@ public function lazyloadBackgroundImages( $html, $buffer ) {
 				continue;
 			}
 
-			$url['url'] = trim( $url['url'], '\'" ' );
+            $url['url'] = esc_url( trim( html_entity_decode($url['url'], ENT_QUOTES), '\'" ' ) );
```

Comment:
@engahmeds3ed try without `html_entity_decode`. What happens?

Reply:
when not using this function the following happens:-
The div is removed completely from the html. when using that

Case 1:
`<div style="background-image: url(&#39;https://www.example.com/wp-content/uploads/2018/04/&#39;telephonie-ip-couleur.jpg&#39;);" class="image_article bkgd-img"></div>`

Case 2
`<div style="background-image: url(&quot;https://www.example.com/wp-content/uploads/2018/04/&#39;telephonie-ip-couleur.jpg&quot;);" class="image_article bkgd-img"></div>`

Reply:
Not sure if this is a valid case, but could we have a situation when the url starts with `&#39;` and ends with `&quot;` ? Can this be added to tests and see if it pass or not?

Reply:
>when not using this function the following happens:-

What does the URL look like before it gets to this line of code? I'm wondering if the quotes are being encoded _before_ it comes to this method, meaning something somewhere else has encoded them. Here, we are decoding them before insertion.

Reply:
@crystinutzaa I will add this to tests and check but I think it will pass without any problem.
@hellofromtonya the url is the same like in the source
`#39;https://www.example.com/wp-content/uploads/2018/04/&#39;telephonie-ip-couleur.jpg&#39;`
so it's the same as input url without being encoded with anything on its way as I think.
So if we passed not encoded quotes or double quotes it won't be encoded.

Reply:
@crystinutzaa New test case added based on your comment and it passed without any problem.